packages/core/.storybook/main.ts

```diff
@@ -34,9 +34,6 @@ const config: StorybookConfig = {
     name: "@storybook/react-webpack5",
     options: {}
   },
-  docs: {
-    autodocs: false
```

Comment:
default is not `true`, default is `tags`, meaning only stories which define `tags: ["autodocs"]` would get autodocs page. It's good for internal components to get a "free" docs page

Reply:
You mean `autodocs: 'tag'` no?